include/swift/ABI/MetadataValues.h

```diff
@@ -130,7 +130,9 @@ enum class ProtocolConformanceTypeKind : unsigned {
   /// platforms, the class object always is the type metadata.
   UniqueDirectClass = 0xF,
 };
-  
+
+typedef TypeMetadataRecordKind ProtocolConformanceTypeKind;
```

Comment:
Instead of the typedef here, can we update uses of `ProtocolConformanceTypeKind` to use the new name?